From c62475329e5e4c4c3f9570546e2b843288f48c61 Mon Sep 17 00:00:00 2001
From: Cory Fields <cory-nospam-@coryfields.com>
Date: Fri, 10 Mar 2017 16:53:03 -0500
Subject: [PATCH] depends: fix zlib build on osx

zlib is sneaky and expects ar to be libtool on darwin.
---
 depends/packages/zlib.mk | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/depends/packages/zlib.mk b/depends/packages/zlib.mk
index 7ff5d00bbdc64..589490800f8bd 100644
--- a/depends/packages/zlib.mk
+++ b/depends/packages/zlib.mk
@@ -7,8 +7,10 @@ $(package)_sha256_hash=c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca11
 define $(package)_set_vars
 $(package)_build_opts= CC="$($(package)_cc)"
 $(package)_build_opts+=CFLAGS="$($(package)_cflags) $($(package)_cppflags) -fPIC"
-$(package)_build_opts+=AR="$($(package)_ar)"
 $(package)_build_opts+=RANLIB="$($(package)_ranlib)"
+$(package)_build_opts+=AR="$($(package)_ar)"
+$(package)_build_opts_darwin+=AR="$($(package)_libtool)"
+$(package)_build_opts_darwin+=ARFLAGS="-o"
 endef
 
 define $(package)_config_cmds
